GPS.Service/Conrollers/ManageOutagesController.cs

```diff
@@ -0,0 +1,79 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+namespace GPS.Service.Conrollers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class ManageOutagesController : ControllerBase
+    {
+        public static IWebHostEnvironment _environment;
+
+        public ManageOutagesController(IWebHostEnvironment environment)
+        {
+            _environment = environment;
+        }
+        
+        public class FileUpload
+        {
+            public IFormFile File { get; set; }
+        }
+        
+        [HttpPost]
+        public async Task<string> Post([FromForm] FileUpload file)
```

Comment:
In your controller methods, you should return Task<IActionResult>, that way you can take advantage of the build in macros for error codes:  Ok, BadRequest, NotFound, etc.  Rather than just a message, we want to return common http request error codes (they may additionally have other data returned including a message)